src/main/resources/jenkins/security/plugins/ldap/LDAPConfiguration/help-ignoreIfUnavailable.html

```diff
@@ -0,0 +1,11 @@
+<div>
+    <p>
```

Comment:
quite weird to not close the `p` tag but seems to case in each help file in the project. I would use `<br />` instead personnally to be valid with the standard.

Reply:
Yeah I had `</p>` tags at first but removed them to be consistent across the plugin. I went ahead and just added them back in 5c1c278.